cypress/fixtures/example.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Want to get rid of this file? Doesn't seem necessary to include

Reply:
I added this back because cypress keeps generating an example with more junk, I think an empty object is a better in-between

Reply:
We could also add it to the gitignore, but we might get confused if we do try to add fixtures later

Reply:
Oh ok got it, that's fine